### PR TITLE
Brantron deploy fix

### PR DESCRIPTION
**Pull request by Brantron**
I believe this is the minimal changeset to get us to production. Locking the gem versions (which was probably the problem) and removing active_storage since it dies without aws keys and we haven't configured them.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -19,6 +19,11 @@ jobs:
     steps:
       - checkout
 
+      - run:
+          name: install dependencies
+          command: |
+            gem install bundler -v 2.0.1
```
Comment (jackiekircher) [inline .circleci/config.yml]: no wonder I couldn't track this down - this direction would have been like, plan D

File: config/environments/production.rb [hunk 2]
```diff
@@ -39,7 +39,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Store uploaded files on the local file system (see config/storage.yml for options)
-  config.active_storage.service = :amazon
+  # config.active_storage.service = :local
```
Comment (maebeale) [inline config/environments/production.rb]: did you mean to remove amazon from production active_storage, @Brantron ?

File: Gemfile [hunk 3]
```diff
@@ -11,15 +11,15 @@ gem 'jbuilder', '~> 2.5'
...
 
 gem 'sassc-rails'
 gem 'turbolinks', '~> 5'
 gem 'uglifier', '>= 1.3.0'
-gem 'webpacker'
+gem 'webpacker', '4.0.0'
 gem 'bootstrap', '~> 4.3.1'
 
-gem "aws-sdk-s3", require: false
+# gem "aws-sdk-s3", require: false
```
Comment (maebeale) [inline Gemfile]: did you mean to remove amazon s3 gem, @Brantron ?

File: config/environments/production.rb (hunk 2 above)
Comment (Brantron) [reply config/environments/production.rb]: no, I can undo that part, it was part of my testing when deploys were failing on active storage

File: Gemfile (hunk 3 above)
Comment (Brantron) [reply Gemfile]: yes, it was failing the build since we don't have creds

File: config/environments/production.rb (hunk 2 above)
Comment (maebeale) [reply config/environments/production.rb]: it's ok. i'll go back through it. aws keys should've been in the encrypted file, but prob good to revisit that whole segment. thanks!

**Comment by maebeale** [comment]
woo hoo!!!

**Pull request merged**